alvr/server_driver/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = 'alvr_server_driver'
-version = '12.1.0'
+version = "12.1.0"
```

Comment:
Should we always use double quotes for all strings in Cargo.toml? I see that I'm the only one that uses single quotes and I regret the choice a bit. My reasoning was that for something like `[target.'cfg(target_os = "android")'.dependencies]` the outermost quotes must be single (to not conflict with the innermost double quotes).

Reply:
I don't have control over which quotes are used for modified values, so I think using double quotes for values would be good for consistency.

Reply:
Ok! I will do the change whenever I see there is the least risk for commit conflicts.